src/transformers/integrations/flash_paged.py

```diff
@@ -3,12 +3,13 @@
 from ..generation.continuous_batching import PagedAttentionCache
 from ..utils import is_flash_attn_2_available
 
+from kernels import get_kernel
```

Comment:
tests are probably failing because of this you need to protect with `is_kernel_available` !